src/ethereum/osaka/vm/instructions/system.py

```diff
@@ -1271,3 +1379,58 @@ def return_contract(evm: Evm) -> None:
 
     # PROGRAM COUNTER
     evm.running = False
+
+
+def pay(evm: Evm) -> None:
+    """
+    Transfer ether to an account.
+
+    Parameters
+    ----------
+    evm :
+        The current EVM frame.
+    """
+    # STACK
+    try:
+        to = to_address_without_mask(pop(evm.stack))
+    except ValueError as e:
+        raise ExceptionalHalt from e
```

Comment:
This error cannot happen in EVM. The `to_address_without_mask` will only throw if the value is larger than an uint256 and pushing such value is not possible in EVM. I think what was intended to do here is to check for the `bytes20`/`uint160` case, so it should be verified that the stack value fits in an `uint160` or `bytes20`. If this is not the case then throw (EVM should internally run like the INVALID opcode)

Reply:
I think the name given to the variable is slightly misleading `MAX_ADDRESS_U256` but it is in fact checking for address that have more than 20 bytes. `b"\xff" * 20`. I will change the variable name to something more appropriate. Thanks for pointing this out.